python/GafferSceneUI/LightEditor.py

```diff
@@ -440,6 +440,37 @@ def __toggleBoolean( self, inspections ) :
 
 		return True
 
+	def __activeInspectionTweaks( self, pathListing ) :
+
+		selection = pathListing.getSelection()
+
+		columns = pathListing.getColumns()
+
+		tweaks = []
+
+		with Gaffer.Context( self.getContext() ) as context :
+			for i in range( 0, len( columns ) ) :
+				column = columns[ i ]
+				if not isinstance( column, _GafferSceneUI._LightEditorInspectorColumn ) :
+					continue
+				for path in selection[i].paths() :
+					context["scene:path"] = GafferScene.ScenePlug.stringToPath( path )
+					inspection = column.inspector().inspect()
+					if inspection is not None :
+						edit = inspection.acquireEdit()
```

Comment:
Nooo! Calling `acquireEdit()` creates an edit, so merely by showing the popup menu with the "Remove Edits..." item in it, we are now _creating_ edits (albeit disabled)! Plus, `acquireEdit()` will throw if the thing isn't editable...

I _think_ we'll be OK if we first check `inspection.sourceType() == SourceType.EditScope` and `inspection.editable()`, but you might want to verify for yourself that there's no situation in which that will end up creating an edit.

Reply:
Fixed in 000eb3ff824f7e2c8f7adf3fecb7e3ac5df9ff35